### DISCUSSION TITLE
Pin to Paper version (not build)

### DISCUSSION THREAD

**Discussion by ja2ui0** [itzg/docker-minecraft-server#1993] [Q&A] [answered]
As Paper build numbers increment from 1 for each version, I understand how to pin 1.19.x based on build number, but what if I want to stay on 1.19.x after 1.20 is released and update on my own schedule? My servers are just for friends, and I typically don't update versions until most / all client and server plugins are supported on new majors which can sometimes take a few weeks.

I'm probably missing something but it doesn't appear obvious in the docs how to pin to a major release.

**Comment by itzg** [comment, marked as answer]
Setting VERSION to "1.19.3" for example will pin that but still allow paper build versions to increment.
